src/openrct2-ui/windows/RideConstruction.cpp

```diff
@@ -1933,14 +1933,15 @@ static void window_ride_construction_mouseup_demolish(rct_window* w)
         _currentTrackPieceType, 0,
         { _currentTrackBegin.x, _currentTrackBegin.y, _currentTrackBegin.z, _currentTrackPieceDirection });
 
+    // FIXME: This can crash if this is multiplayer, the window might be closed before the callback fires.
```

Comment:
Assume fix is for the game action result to specify the ride id?

Reply:
You could just capture the ride id before and find the window in the callback via id.

Reply:
I can get rid of the comment in this PR, I just didn't wanna loose track of it.